script.js

```diff
@@ -1,4 +1,55 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computerNum = randomNumber();
+  var computerChoice = "null";
+  var myOutputValue = "null";
+
+  if (computerNum == 1) {
```

Comment:
Would be cleaner if the scissors paper stone result is extracted into another function. 

For example you can create another function that returns scissors, paper or stone.  
You'll then call this function here. 

Let me know if this makes sense :)